ledger_ed25519.go

```diff
@@ -0,0 +1,156 @@
+package crypto
+
+import (
+	"fmt"
+	"github.com/pkg/errors"
+
+	// "github.com/tendermint/ed25519"
+	ledger "github.com/zondax/ledger-goclient"
+)
+
+func pubkeyLedgerEd25519(device *ledger.Ledger, path DerivationPath) (pub PubKey, err error) {
+	key, err := device.GetPublicKeyED25519(path)
+	if err != nil {
+		return pub, fmt.Errorf("Error fetching public key: %v", err)
```

Comment:
`pub` will be nil in this case. Make this explicit by writing `return nil, fmt.Errof(...)`. Also, error messages are usually not capitalized (as the error might bubble up and wrapped into another error msg).